src/internal/modules/gauging-stations/lib/helpers.js

```diff
@@ -136,6 +137,92 @@ const handlePost = async request => {
   return services.water.gaugingStations.postLicenceLinkage(gaugingStationId, licenceId, set(parsedPayload, 'licenceVersionPurposeConditionId', null));
 };
 
+const handleRemovePost = async request => {
+  const sessionData = session.get(request);
+  let promises = [];
+  if (sessionData.selectedCondition) {
+    if (sessionData.selectedCondition.value) {
+      sessionData.selectedLicence = null;
+      const selectArr = sessionData.selectedCondition.value;
+      promises = selectArr.map(licenceGaugingStationId => services.water.gaugingStations.postLicenceLinkageRemove(licenceGaugingStationId));
+    }
+  } else if (sessionData.selectedLicence && sessionData.selectedLicence.value) {
+    const arrayOfLicenceGaugingStationsRecords = sessionData.selectedLicence.options.choices;
+    promises = arrayOfLicenceGaugingStationsRecords.map(row => services.water.gaugingStations.postLicenceLinkageRemove(row.licenceGaugingStationId));
+  }
+  return Promise.all(promises);
+};
+
+const humaniseAlertType = str => {
+  str = str.replace('stop_or_reduce', 'Reduce');
+  str = str.replace('stop', 'Stop');
+  str = str.replace('reduce', 'Reduce');
+  return str;
+};
+
+const humaniseUnits = str => {
+  str = str.replace('gal', 'Gallons');
+  str = str.replace('Ml', 'Megalitres');
+  str = str.replace('m³', 'Cubic metres');
+  str = str.replace('l/', 'Litres/');
+  str = str.replace('/d', ' per day');
```

Comment:
Both of these have the slashes included so the expected value would have to be `l//d`

Reply:
A generalized longForm function splits up string into words and expand abbreviations.